add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
+    $year = $_POST['year'];
+
+    $query = "INSERT INTO books (title, author, year) VALUES ('$title', '$author', $year)";
+    if (mysqli_query($connection, $query)) {
+        echo "Book added successfully!";
+    } else {
```

Comment:
Observations: HTML and PHP code are mixed, making it hard to maintain and test.
Recommendations: Separate logic from presentation by using a templating engine like Twig or Blade, or at least separate PHP logic into a different file or function.